lib/typo_killer/finder.ex

```diff
@@ -11,12 +11,18 @@ defmodule TypoKiller.Finder do
 
   def find_typos({words, dictionary}) do
     words
-    |> Enum.map(&Task.async(fn -> calculate_distance(&1, dictionary) end))
-    |> Enum.flat_map(&Task.await(&1, :infinity))
+    |> Flow.from_enumerable()
+    |> Flow.map(fn word -> calculate_distance(word, dictionary) end)
+    |> Flow.reduce(fn -> MapSet.new() end, &merge_partial_result/2)
```

Comment:
I love how simple it is was to move from serialized processing to concurrent processing. 😍